app/src/main/java/org/schabi/newpipe/fragments/list/search/SearchFragment.java

```diff
@@ -486,6 +486,9 @@ private void showSearchOnStart() {
                     + lastSearchedString);
         }
         searchEditText.setText(searchString);
+        if (android.os.Build.VERSION.SDK_INT < android.os.Build.VERSION_CODES.LOLLIPOP) {
+            searchEditText.setHintTextColor(searchEditText.getTextColors().withAlpha(128));
```

Comment:
What's up with these `.withAlpha()` calls?

Reply:
Material Design guidelines say that hint color should be 50% lighter than text color, on Android Lollipop and higher the color you pass to `setHintTextColor` automatically brightens by 50%, but not on KitKat.
`ColorStateList.withAlpha` accepts `int` in range of 0-255, 255 is for 100% and 0 is for 0% opacity.